tests/integration_tests/lightgbm_tuner_tests/test_optimize.py

```diff
@@ -352,10 +353,15 @@ def test__parse_args_wrapper_args(self) -> None:
         [("auc", "maximize", -np.inf), ("mse", "minimize", np.inf)],
     )
     def test_best_score(self, metric: str, study_direction: str, expected: float) -> None:
-        with turnoff_train(metric=metric):
+        lgbm_params = {}
+        lgbm_params["metric"] = metric
+        _handling_alias_metrics(lgbm_params)
+        with turnoff_train(metric=lgbm_params["metric"]):
             study = optuna.create_study(direction=study_direction)
             runner = self._get_tuner_object(
-                params=dict(lambda_l1=0.0, metric=metric), kwargs_options={}, study=study,
+                params=dict(lambda_l1=0.0, metric=lgbm_params["metric"]),
+                kwargs_options={},
+                study=study,
```

Comment:
Cool! IMO, we have an alternative approach. How about changing the metric name in L353 from `mse` to `l2`? Then, we can keep the existing test code here.

Reply:
I agree. If that is ok I think we should do that. I was not sure if there was a special meaning around the `mse` in that case.